pkg/controller/operators/catalog/operator.go

```diff
@@ -595,8 +604,24 @@ func (o *Operator) syncConfigMap(logger *logrus.Entry, in *v1alpha1.CatalogSourc
 
 	logger.Debug("checking catsrc configmap state")
 
+	var updateLabel bool
 	// Get the catalog source's config map
 	configMap, err := o.lister.CoreV1().ConfigMapLister().ConfigMaps(in.GetNamespace()).Get(in.Spec.ConfigMap)
+	// Attempt to look up the CM via api call if there is a cache miss
+	if k8serrors.IsNotFound(err) {
+		configMap, err = o.opClient.KubernetesInterface().CoreV1().ConfigMaps(in.GetNamespace()).Get(context.TODO(), in.Spec.ConfigMap, metav1.GetOptions{})
+		// Found cm in the cluster, add managed label to configmap
+		if err == nil {
+			labels := configMap.GetLabels()
+			if labels == nil {
+				labels = make(map[string]string)
+			}
+
+			labels[install.OLMManagedLabelKey] = "false"
```

Comment:
I don't see a problem here as I set it to `false` and the other configmaps that we create to be `true`. Just because we use the key only for the filter, it doesn't mean the value doesn't have meaning. I'm fine with this label.